frontend/src/routes/settings/admin/oidc-clients/oidc-client-form.svelte

```diff
@@ -30,15 +30,15 @@
 
 	const client: OidcClientCreate = {
 		name: existingClient?.name || '',
-		callbackURLs: existingClient?.callbackURLs || [''],
+		callbackURLs: existingClient?.callbackURLs || [], // Changed from [''] to []
 		logoutCallbackURLs: existingClient?.logoutCallbackURLs || [],
 		isPublic: existingClient?.isPublic || false,
 		pkceEnabled: existingClient?.pkceEnabled || false
 	};
 
 	const formSchema = z.object({
 		name: z.string().min(2).max(50),
-		callbackURLs: z.array(z.string().nonempty()).nonempty(),
+		callbackURLs: z.array(z.string()).optional().default([]), // Made optional and allow empty strings
```

Comment:
```suggestion
		callbackURLs: z.array(z.string().nonempty()).default([])
```
We still have to make sure that the callback URL isn't empty, if the user add one.